kong/clustering/data_plane.lua

```diff
@@ -62,11 +65,35 @@ function _M.new(parent)
 end
 
 
+local function sort(t)
+  if type(t) == "table" then
+    local i = 0
+    local o = {}
+    for k, v in tablex.sort(t) do
+      o[i+1] = sort(k)
+      o[i+2] = ":"
+      o[i+3] = sort(v)
+      o[i+4] = ";"
+      i=i+4
+    end
+
+    return concat(o, nil, 1, i)
+  end
+
+  return tostring(t)
```

Comment:
Don't encode the number `1` the same as the string `"1"`. Also avoid problems with strings starting with `{` (conflict with table separators)
```suggestion
  if type(t) == "string" then
    return '"' .. t:gsub('"', '\\"') .. '"'
  else
    return tostring(t)
  end
```